src/utils/beneficial.owner.individual.ts

```diff
@@ -74,6 +76,12 @@ export const getBeneficialOwnerIndividualById = (req: Request, res: Response, ne
 
     const appData = getApplicationData(req.session);
 
+    // Redis removal work - Add extra template options if Redis Remove flag is true and on Registration journey
```

Comment:
I would move the line 
const appData = getApplicationData(req.session);
 underneath the new redis removal 'if', as the appData is used by the block of code underneath the redis block.

Reply:
Done